enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/CypherComparisonSupport.scala

```diff
@@ -457,6 +405,8 @@ object CypherComparisonSupport {
 
   object Runtimes {
 
+    val all = Runtimes(CompiledBytecode, CompiledSource, Slotted, Interpreted, ProcedureOrSchema, Default)
```

Comment:
In `Planner.all` and `Versions.all` Alex and I explicitly excluded `Default`, since one of them would be the default anyway. Is that required to be included here?

Reply:
There is no `Versions.all`, and `Default` in runtimes is actually different from specifying one